tests/test_head_loss.py

```diff
@@ -437,7 +439,7 @@ def energy_system_options(self):
                     results["Pipe4.HeatIn.Q"]
                     / (solution.parameters(0)["Pipe4.diameter"] ** 2 / 4.0 * np.pi),
                     0.0,
-                    atol=1e-07,
+                    atol=1e-04,
```

Comment:
When heatlosses are turned off, shouldn't it be even easier to match 0.0 velocity, thus I do not see why the absolute tolerance needs to be larger.

Reply:
I do not know why this happened. I did notice that the heating demand value was unrealistically low, in the order of kW. I haven now increased the heating demand such that it is in the order of MW, then all work as before and the pipe velocity is then very small and the "atol=1e-07" still works.